backend/internal/handlers/webhooks/webhooks.go

```diff
@@ -87,6 +88,14 @@ func (s *WebHookService) PushEvent(c *fiber.Ctx) error {
 	return nil
 }
 
+func hasNamedPusher(pushEvent github.PushEvent) bool {
+	return pushEvent.Pusher != nil && pushEvent.Pusher.Name != nil
+}
+
+func isBotPushEvent(pushEvent github.PushEvent) bool {
+	return strings.Contains(*pushEvent.Pusher.Name, "[bot]")
```

Comment:
I think requiring this githubuser.Name is fine because the bot will always have a name, and if it doesn't then it must not be a bot